docs/rest-api/source/includes/wp-api-v3/deposits.md

```diff
@@ -0,0 +1,575 @@
+# Deposits
+
+The Deposits API endpoints provide access to an account's deposits data, including an overview of account balances, deposit schedule and deposit history.
+
+## Get deposits overview for all account deposit currencies
+
+Fetch an overview of account deposits for all deposit currencies. This includes details for the last paid deposit, next scheduled deposit, and last manual deposits.
+
+### HTTP request
+
+<div class="api-endpoint">
+	<div class="endpoint-data">
+		<i class="label label-get">GET</i>
+		<h6>/wp-json/wc/v3/payments/deposits/overview-all</h6>
+	</div>
+</div>
+
+### Returns
+
+-   `deposit` _object_
+    -   `last_paid` _array_ - The last deposit that has been paid for each deposit currency.
+    -   `next_scheduled` _array_ - The next scheduled deposit for each deposit currency.
+    -   `last_manual_deposits` _array_ - Manual deposits that have been paid in the last 24 hours.
+-   `balance` _object_
+    -   `pending` _array_ - The pending balance for each deposit currency.
+        -   `amount` _int_ - The amount of the balance.
+        -   `currency` _string_ - The currency of the balance. E.g. `usd`.
+        -   `source_types` _object_ - The amount of the balance from each source type, e.g. `card` or `financing`.
```

Comment:
How strict are we. For example, [Stripe's docs](https://stripe.com/docs/api/balance/balance_object#balance_object-pending-source_types) mentions that `source_types` is nullable.

Reply:
The closer to reality we are, the better! Helps us consume downstream with confidence.

I'll double-check these properties against the Stripe API where relevant.